operator/docs/lokistack/sop.md

```diff
@@ -46,6 +46,36 @@ A service(s) is failing to process at least 10% of all incoming requests.
     - `loki_ingester_wal_disk_full_failures_total`
     - `loki_ingester_wal_corruptions_total`
 
+## Lokistack Gateway Request Errors
+
+### Impact
+
+The Lokistack Gateway component is unable to perform its duties for a number of requests, resulting in potential loss of data.
+
+### Summary
+
+The Lokistack Gateway is failing to process at least 10% of all incoming requests.
+
+### Severity
+
+`Critical`
+
+### Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging`
+    - `openshift-operators-redhat`
+
+### Steps
+
+- Ensure that the deployed Loki components are ready and available
+- Ensure that store services (`ingester`, `querier`, `index-gateway`, `compactor`) can communicate with backend storage
+- Examine metrics for signs of failure
+  - WAL Complications
+    - `loki_ingester_wal_disk_full_failures_total`
+    - `loki_ingester_wal_corruptions_total`
```

Comment:
We need a step for the user to check that the gateway containers are ready and available, e.g. maybe opa-openshift just fails to talk to the apiserver.